src/cli/build.rs

```diff
@@ -0,0 +1,30 @@
+use clap::Args;
+
+use super::{Handler, Result};
+
+/// Arguments for `BuildCommand`
+///
+/// Usage :
+/// `quark build --image <container_image_url> [--offline] --quardle <quardle_name>`
+#[derive(Debug, Args)]
+pub struct BuildCommand {
+    /// The name of the generated quardle, with the suffix `.qrk`
+    #[clap(short, long)]
+    quardle: String,
+
+    /// The container image url to use
+    #[clap(short, long)]
+    image: String,
+
+    /// Indicates wether or not the container image is bundled into the initramfs image
```

Comment:
What happens if it's not. Where will the image come from?

Reply:
This should be the role of Kaps to pull the image from the name / URL stated in the build command (and in the `quark.json` file).